### PR TITLE
add WaveRNN vocoder

### PR DESCRIPTION
**Pull request by pppku**

### CODE REVIEW

**Comment by pppku** [comment]
This change still need more test

**Review by ftshijt — COMMENTED** [review]
do not add pyt file to the repo. I will merge this pr after you remove that.

File: SVS/model/infer.py [hunk 1]
```diff
@@ -285,6 +287,29 @@ def infer(args):
...
+            feat_dims=80,
+            compute_dims=128,
+            res_out_dims=128,
+            res_blocks=10,
+            hop_length=275,  # 12.5ms - in line with Tacotron 2 paper
+            sample_rate=22050,
+            mode="MOL",
+        ).to(device)
+
+        voc_model.load("./weights/wavernn/latest_weights.pyt")
```
Comment (ftshijt) [inline SVS/model/infer.py:311]: might be better to use an argument and pass it to here

**Comment by ftshijt** [comment]
I will merge the PR for now because it is urgent. But please resolve the problem I mentioned above. For the model sharing, I recommend you upload it to some open-website or storage (e.g., google drive) so that we could get the pre-trained model online.

**Comment by pppku** [comment]
copy that

**Pull request merged**